apps/frontend/core-dapp/src/stores/BalancerStore.ts

```diff
@@ -53,6 +52,25 @@ export class BalancerStore {
     )
   }
 
+  get defaultWstEthToWethRate(): BigNumber | undefined {
+    if (this.wstEthWethPoolData === undefined) return undefined
+    const amount = parseEther('1')
+    const outputOut = exactTokenInForTokenOut({
+      ...this.wstEthWethPoolData,
+      tokenIndexIn: 0,
+      tokenIndexOut: 1,
+      amount,
+    })
+
+    // calculate rate for 1 wstETH
+    return outputOut.mul(parseEther('1')).div(amount)
+  }
+
+  get defaultWethToWstEthRate(): BigNumber | undefined {
```

Comment:
```suggestion
  get defaultEthToWstEthRate(): BigNumber | undefined {
```

I don't think we refer to ETH as WETH anywhere else, right?

Reply:
ahh probably have to fix this up separately, I did find a few things that were using Weth, but will have a separate PR to refactor after we fix this issue